src/Description.js

```diff
@@ -1,15 +1,45 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './Description.css';
+import Poster from './Poster';
+import Button from './Button';
 
-function Description({title, rating, summary, releaseDate}) {
-  return (
-    <div>
-    <h2>{title}</h2>
-    <p>{releaseDate}</p>
-    <p>{summary}</p>
-    <h3>{rating.toFixed(1)}  ⭐️</h3>
-    </div>
-  )
+class Description extends Component {
+  constructor(){
+    super()
+    this.state = {
```

Comment:
We should add an error: false to our state here, and add a conditional render with an error message, since we have the catch set up already in our fetch call.